rest-api/etl/setup_local_database_for_etl.sh

```diff
@@ -82,22 +87,27 @@ rm -rf ${CSV_DIR}
 mkdir -p ${CSV_DIR}
 mkdir -p ${CSV_DIR}/cdm
 mkdir -p ${CSV_DIR}/voc
+if [ "${GENERATE_SQL_DUMP}" ]
+then
+  mkdir -p ${OUTPUT_DIR}
+  chmod -R 0777 ${OUTPUT_DIR}
+fi
 
-SERVICE_ACCOUNT=pmi-drc-api-test@appspot.gserviceaccount.com
+PROJECT=pmi-drc-api-test
+CREDS_ACCOUNT="${ACCOUNT}"
 
 echo "Activating service account..."
-#gcloud iam service-accounts keys create $CREDS_FILE --iam-account=$SERVICE_ACCOUNT --account=$ACCOUNT
-#gcloud auth activate-service-account pmi-drc-api-test@appspot.gserviceaccount.com --key-file=$CREDS_FILE
+source tools/auth_setup.sh
+gcloud auth activate-service-account circle-deploy@all-of-us-rdr-staging.iam.gserviceaccount.com --key-file=$CREDS_FILE
```

Comment:
Same comment on hardcoding, especially given that project appears to be from "test" and SA is "staging"

Reply:
See above.